### ISSUE TITLE
Copying a non-existing file copies the mft in its place?

### ISSUE THREAD

**Issue by lespea** [jschicht/RawCopy#17]
If you run the following command, even though the source file does not exist RawCopy just warns you and then starts copying a file anyway to the destination.  Looking at what's created, it appears to be the MFT file (not confirmed).  Looks like this started happening on version 1.0.0.13 (1.0.0.12 works as expected).

```
rawCopy64.exe /FileNamePath:"c:\abcdefg" /OutputPath:"c:\tmp\"

Warning: File not found with regular file search: c:\abcdefg
Writing: abcdefg
```

**Comment by thepaulbenoit** [comment]
I get this only when I use variables in /FileNamePath

**Comment by 0xpaulbenoit** [comment]
it messes up if a file doesn't exist and has \ in the file path:
	`/FilepathName:c:noexist`			will correctly say it doesn't exist
	`/FilepathName:%systemdrive%noexist`	will correctly say it doesn't exist
	`/FilepathName:c:\noexist`		will erroneously copy the $MFT
	`/FilepathName:c:noexist\test.txt`	will erroneously copy the $MFT

**Comment by lespea** [comment]
Good catch

**Comment by ruchirarya** [comment]
@jschicht Any information on this open bug?

I wish this great tool was an open source project and probably DFIR community would have contributed a lot more features to this tool.

**Comment by jschicht** [comment]
Bug is resolved in version 1.0.0.22.
